docs/source/paper.md

```diff
@@ -28,9 +28,13 @@ bibliography: paper.bib
 
 The computational sciences have seen a huge increase in the use of high-throughput, automated, workflows over the course of the last two decades or so.  Focusing on just our domain of computational materials science, there have been several large scale initiatives to provide high-quality results from standardised calculations [@Landis2012; @Curtarolo2012; @Jain2013; @Saal2013; @Draxl2019a; @Talirz2020].  Almost all of these repositories are populated using results from high-throughput quantum mechanical calculations that rely on workflow frameworks [@Jain2015a; @Mortensen2020], including our own [AiiDA](http://www.aiida.net/) [@Pizzi2016; @Huber2020] which powers the [Materials Cloud](https://www.materialscloud.org/).  One of the many challenges for such frameworks is maximising fault-tolerance whilst simultaneously maintaining high-throughput, often across several systems (typically the client launching the tasks, the supercomputer carrying out the computations and the server hosting the database).
 
-On the software level these problems are perhaps best addressed by using messaging brokers that take responsibility for guaranteeing the durability and atomicity of messages and often enable event based communication.  Indeed, solutions such as [RabbitMQ](https://www.rabbitmq.com/) see widespread adoption in industry, however, adoption in academia has been more limited, with home-made queue data structures, race condition susceptible locks and polling based solutions being commonplace. This is likely due to message brokers typically having complex APIs (which reflect the non-trivial nature of the underlying protocol) as well as the lack of familiarity with event-based programming in general within the community.  [KiwiPy](https://kiwipy.readthedocs.io/en/latest/) was designed specifically to address both these issues, by providing a tool that allows building robust event based systems with an interface that is as simple as possible.
+On the software level these problems are perhaps best addressed by using messaging brokers that take responsibility for guaranteeing the durability and atomicity of messages and often enable event-based communication.  Indeed, solutions such as [RabbitMQ](https://www.rabbitmq.com/) see widespread adoption in industry, however, adoption in academia has been more limited, with home-made queue data structures, race condition susceptible locks and polling based solutions being commonplace. This is likely due to message brokers typically having complex APIs (which reflect the non-trivial nature of the underlying protocol) as well as the lack of familiarity with event-based programming in general within the community.  [KiwiPy](https://kiwipy.readthedocs.io/en/latest/) was designed specifically to address both these issues, by providing a tool that allows building robust, event-based systems with an interface that is as simple as possible.
```

Comment:
This is a good example why it is useful to maintain one-sentence per line in git projects, even for text. Right now, the entire block is marked as changed, even though normally on single sentences, you get the changed words highlighted. This makes it a lot easier to see the differences and to review them.

Reply:
Thanks.  I tried to have them on separate lines but MD kept interpreting the newline literally and I coudln't figure out why until now.  Didn't know that if you have two spaces (my usual) at the end of the sentence than a new line will be interpreted literally.